packages/components/src/mobile/picker/styles.native.scss

```diff
@@ -19,3 +19,11 @@
 	padding-left: 0;
 	padding-right: 0;
 }
+
+.components-picker__button-title {
+	color: $blue-50;
+}
+
+.components-picker__button-title--dark {
+	color: $blue-30;
+}
```

Comment:
These static colors make sense to me for the current goal — a consistent color within the editor. Long term, I would imagine we should revert this change so that the host app once again can control the picker colors.